cl2-component-library/src/components/Select/index.tsx

```diff
@@ -24,26 +24,26 @@ const Arrow = styled(Icon)`
   `}
 `;
 
-const Container = styled.div`
+export const SelectContainer = styled.div`

```

Comment:
```suggestion
const SelectContainer = styled.div`
```

This one doesn't strictly need to be exported

Reply:
But also doesn't hurt I guess